test/api/test_invoice.py

```diff
@@ -447,3 +482,153 @@ def test_retrieve_invoice(self, mock_requests):
         self.assertTrue(isinstance(result, Invoice))
 
         self.assertEqual(result.uuid, "inv_22910fc6-c931-48e7-ac12-90d2cb5f0059")
+
+    @requests_mock.mock()
+    def test_retrieve_invoice_with_validation_type(self, mock_requests):
+        mock_requests.register_uri(
+            "GET",
+            ("https://api.chartmogul.com/v1/invoices/inv_22910fc6-c931-48e7-ac12-90d2cb5f0059"
+             "?validation_type=all"),
+            request_headers={"Authorization": "Basic dG9rZW46"},
+            headers={"Content-Type": "application/json"},
+            status_code=200,
+            json=retrieveInvoiceExample,
+        )
+
+        config = Config("token")  # is actually checked in mock
```

Comment:
I don't understand the comment - what is checked in mock and what does checking in mock mean? Please clarify or remove.